tools/lsvast/lsvast.cpp

```diff
@@ -142,10 +142,15 @@ caf::expected<Kind> classify(const std::filesystem::path& path) {
   if (err)
     return caf::make_error(vast::ec::filesystem_error,
                            "Invalid path: ", err.message());
-
-  if (is_directory && path.stem() == "vast.db")
```

Comment:
Did I accidentally introduce this bug when switching to `std::filesystem` here? 🤔 

Sorry if I did!

Reply:
You didn't, it's been broken for quite a while apparently. This is an internal debugging tool, so it's not really covered by tests or CI. We usually just work on it when we need it.

Reply:
Got it; that's what I figured when I was rewriting parts of it to get off of `vast::path`. Thanks for confirming.